bin/runtime/src/lib.rs

```diff
@@ -832,6 +833,24 @@ pub type SignedExtra = (
     frame_system::CheckWeight<Runtime>,
     pallet_transaction_payment::ChargeTransactionPayment<Runtime>,
 );
+
+pub struct NominationPoolsMigrationV5OldPallet;
```

Comment:
nit: rename to smth like `MaxGlobalCommision`

Reply:
done